packages/clerk-js/src/ui/elements/PasswordInput.tsx

```diff
@@ -19,8 +20,10 @@ export const PasswordInput = (props: PropsOfComponent<typeof Input>) => {
         type={hidden ? 'password' : 'text'}
         sx={theme => ({ paddingRight: theme.space.$8 })}
       />
-      <Button
+      <IconButton
         elementDescriptor={descriptors.formFieldInputShowPasswordButton}
+        iconElementDescriptor={descriptors.formFieldInputShowPasswordIcon}
+        aria-label={`${hidden ? 'Show' : 'Hide'}password'`}
```

Comment:
❓ Something is up with the quotes, right?

Reply:
I did forget a space there :), but what do you mean?

Reply:
Ah yes, the last `'` shouldn't be there as well. Thanks. Doesn't cause any error so I missed it. My bad.